.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018-present Sonatype Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: 2.1
+jobs:
+  release:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - run: curl -sL https://git.io/goreleaser | bash
+  build:
+    docker:
+      - image: circleci/golang:1.14
+        environment:
+          GO111MODULE: "on"
+          TEST_RESULTS: /tmp/test-results
+    working_directory: /go/src/github.com/sonatype-nexus-community/ahab
+    steps:
+      - checkout
+      - run: mkdir -p $TEST_RESULTS/gotest
+      - restore_cache:
+          keys:
+            - go-mod-v1-{{ checksum "go.sum" }}
+      - run: go get -u github.com/jstemmer/go-junit-report
+      - run: make deps
+      - run: make ci-lint
+      - run: make test | go-junit-report > $TEST_RESULTS/gotest/report.xml
+      - save_cache:
+          key: go-mod-v1-{{ checksum "go.sum" }}
+          paths:
+            - "/go/pkg/mod"
+      - store_artifacts:
+          path: /tmp/test-results/
+      - store_test_results:
+          path: /tmp/test-results/
+
+workflows:
+  version: 2
+  build_and_release:
+    jobs:
+      - build
+      - release:
+          context: ahab
```

Comment:
Yeah, more or less just carving out a new name, so it'll break if this is merged and not setup (rather than use something elses and then break later when we change something on accident)